examples/Matlab/MR/acquisition_data.m

```diff
@@ -36,8 +36,8 @@ function acquisition_data(engine)
 % all acquisition data generated by this script will be stored in memory
 % (the input data will remain in the input file)
 AD = MR.AcquisitionData();
-scheme = AD.get_storage_scheme();
-AD.set_storage_scheme('memory');
+%scheme = AD.get_storage_scheme();
+%AD.set_storage_scheme('memory');
```

Comment:
I always comment out before deleting in case I delete something that actually should have stayed and do not notice it until much later, when restoring could be a lot of hassle. All these commented out lines will go before merge.